src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="conteiner">
+      <div class="conteiner_numbers conteiner_numbers--month_length">
```

Comment:
please read the instructions to the task
a container with calendar should have 2 modifier classes
`calendar--start-day--{VALUE GOES HERE}` - with mon, tue, wed, thu, fri, sat and sun values
and `calendar--month-length--{NUMBER OF DAYS GOES HERE}` - with values 28, 29, 30 and 31